comps/lvms/src/integrations/dependency/llava/README.md

```diff
@@ -62,13 +62,13 @@ docker build -t opea/lvm-llava:latest --build-arg https_proxy=$https_proxy --bui
 - Xeon
 
 ```bash
-docker run -p 8399:8399 -e http_proxy=$http_proxy --ipc=host -e https_proxy=$https_proxy opea/lvm-llava:latest
+docker run -d --name lvm-llava -p 8399:8399 -e http_proxy=$http_proxy --ipc=host -e https_proxy=$https_proxy opea/lvm-llava:latest
```

Comment:
To align with https://github.com/opea-project/GenAIComps/blob/main/comps/lvms/deployment/docker_compose/compose.yaml#L7, maybe we change the name from `lvm-llava` to `llava-service` ?